signalfx/resource_signalfx_team_test.go

```diff
@@ -0,0 +1,114 @@
+package signalfx
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	sfx "github.com/signalfx/signalfx-go"
+)
+
+const newTeamConfig = `
+resource "signalfx_team" "myteamXX" {
+    name = "Super Cool Team"
+		description = "Fart noise"
+
+    notifications_critical = [ "Email,test@example.com" ]
+    notifications_default = [ "Webhook,,,https://www.example.com" ]
```

Comment:
LGTM. What docs do I reference to see what values should go here?

Reply:
[Here](https://www.terraform.io/docs/providers/signalfx/r/detector.html#notification-format)!